torch/csrc/autograd/python_variable.cpp

```diff
@@ -1354,9 +1354,11 @@ static int THPVariable_subclass_traverse(
       // Note that this is only valid as long as user don't share non-owning references
       // across different threads (which is crazy and should never be done).
 
-      if (tensor.use_count() == 1) {
```

Comment:
In our case, tensor.use_count() will actually be 2 because a reference is held by the SavedVariable.
If that's the case, we do want to traverse